physcraper/__init__.py

```diff
@@ -2390,156 +2261,91 @@ def remove_identical_seqs(self):
         tmp_dict = dict((taxon.label, self.data.aln[taxon].symbols_as_string()) for taxon in self.data.aln)
         old_seqs = tmp_dict.keys()
         # Adding seqs that are different, but needs to be maintained as diff than aln that the tree has been run on
-        # need to re-calculate orig_seq_len before using it
-        self.data.orig_seqlen = [len(self.data.aln[tax].symbols_as_string().replace("-", "").replace("N", "")) for tax in
-                                 self.data.aln]
+        if self.data.orig_seqlen == []:
+            self.data.orig_seqlen = [len(self.data.aln[tax].symbols_as_string().replace("-", "").replace("N", "")) for tax in self.data.aln]
```

Comment:
I added this line to update the original sequence length. Originally, the only time it was calculated was in prune_short() before removing short sequences, so it did have a value, but that one was not up to date.

Reply:
Ah right - but if this gets updated on each round this represents the current sequence length rather than the original sequence length? I'm not sure which one we really want here.

Reply:
Even without this line, the orig_seqlen is updated in every prune_short() function call. It's just not up to date with the alignment being used here.